blueoil/datasets/pascalvoc_custom.py

```diff
@@ -0,0 +1,74 @@
+# Copyright 2020 The Blueoil Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+import os
+import os.path
+
+from blueoil.datasets.pascalvoc_base import PascalvocBase
+
+
+class PascalVOCCustom(PascalvocBase):
+    classes = []
+    available_subsets = []
+    extend_dir = ""
+
+    def __init__(
+            self,
+            *args,
+            **kwargs
+    ):
+        PascalVOCCustom.classes = kwargs['classes']
+        PascalVOCCustom.available_subsets = kwargs['available_subsets']
+        PascalVOCCustom.extend_dir = kwargs['extend_dir']
+        super().__init__(
+            *args,
+            **kwargs,
+        )
```

Comment:
Let's keep the code short.
```suggestion
        super().__init__(*args, **kwargs)
```

Reply:
Fix.